cli/command/container/utils.go

```diff
@@ -75,7 +75,7 @@ func legacyWaitExitOrRemoved(ctx context.Context, apiClient client.APIClient, co
 
 	eventProcessor := func(e events.Message) bool {
 		stopProcessing := false
-		switch e.Status {
+		switch e.Action {
 		case "die":
```

Comment:
race condition; was about to push to your branch; I had this, which also replaces these for the consts; https://github.com/docker/cli/pull/6132